### DISCUSSION TITLE
Support new form library with validations

### DISCUSSION THREAD

**Discussion by fabxcp** [primefaces/community#2654] [PrimeVue]
Hello,
looking at the roadmap, I see that in Q3 you plan to release the support for a new form library with valiations.
Could you please indicate when this new version will be released?

**Comment by basitiyidir** [comment]
I noticed that VeeValidate has been quietly deprecated. I'm eagerly waiting for Form Library to be released in my new project. However, if it doesn't come out this month, I plan to use VeeValidate.

**Reply by fabxcp** [reply]
Thanks @basitiyidir . Do you know when the new Form library will be released?

**Reply by basitiyidir** [reply]
According to what's written here and if they stick to this schedule, it should be completed by the end of September.

https://primevue.org/roadmap/
